deployer/deployer-services/src/main/java/io/hyscale/deployer/services/handler/impl/V1DeploymentHandler.java

```diff
@@ -389,8 +389,12 @@ public String getPodSelector(ApiClient apiClient, String selector, boolean label
                 .get(K8SRuntimeConstants.K8s_DEPLOYMENT_POD_TEMPLATE_HASH);
 
         logger.debug("pod-template-hash: {}", podTemplateHash);
-
-        return selector
-                .concat("," + K8SRuntimeConstants.K8s_DEPLOYMENT_POD_TEMPLATE_HASH + "=" + podTemplateHash);
+        return K8SRuntimeConstants.K8s_DEPLOYMENT_POD_TEMPLATE_HASH + "=" + podTemplateHash;
     }
+
+	@Override
+	protected String getPodRevision(V1Deployment t) {
+		// TODO
```

Comment:
You can take the above logic of get(0) deployment to call this getPodRevision(V1Deployment t).